src/components/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPannel from './ButtonPannel';
+
+function App() {
```

Comment:
- [  ] [OPTIONAL] Try to rewrite the components with arrow functions for a beautiful ES6 syntax.